### ISSUE TITLE
Kubernetes - KUBECONFIG line appended to /etc/environment multiple times

### ISSUE THREAD

**Issue by scottyeager** [threefoldtech/tf-images#99]
In the Kubernetes image, the first line of `/scripts/entrypoint.sh` is:

```
echo "KUBECONFIG=/etc/rancher/k3s/k3s.yaml" >> /etc/environment
```

This line can be executed more than once, if the VM is restarted or the `init` job runs more than once for any reason. The result is many copies of the outputted line appended to the environment file.

Better to do this in a oneshot job via zinit or add some conditional logic to only execute once.

**Comment by sameh-farouk** [comment]
Update:
PR ready for review
https://github.com/threefoldtech/tf-images/pull/122

**Comment by sameh-farouk** [comment]
**Update:**
a new Flist is available now with the latest Kubernetes release 1.26 and a few fixes and improvements.
It includes a fix for this issue as we now export KUBECONFIG at bash login.
https://github.com/threefoldtech/tf-images/pull/122
The update will take place, as soon as the new flist gets promoted to the official apps' repo.
